docs/guides/components.md

```diff
@@ -35,30 +35,30 @@ The actual default component structure of the Video.js player looks something li
 
 ```
 Player
```

Comment:
The idea for this section to go with the "class names" rather than the instance names.